.golangci.yaml

```diff
@@ -1,13 +1,37 @@
+version: "2"
 linters:
   enable:
     - gosec
-
+  disable:
+    - staticcheck
+  exclusions:
+    generated: lax
+    presets:
+      - comments
+      - common-false-positives
+      - legacy
+      - std-error-handling
+    rules:
+      - linters:
+          - gosec
+        path: (.+)_test\.go
+    paths:
+      - third_party$
+      - builtin$
+      - examples$
+formatters:
+  exclusions:
+    generated: lax
+    paths:
+      - third_party$
+      - builtin$
+      - examples$
 run:
   timeout: 5m
-
 issues:
   exclude-rules:
     # Skip test files when running the gosec linter.
     - path: (.+)_test\.go
       linters:
         - gosec
```

Comment:
This looks redundant in the current config. It's already set in `linters/exclusions/rules/`. Could you check which config is valid and remove the other one?